i18n/src/main/resources/openfire_i18n.properties

```diff
@@ -2620,9 +2620,6 @@ ssl.settings.client.custom.mutualauth.bosh=Mutual authentication (BOSH connectio
 ssl.settings.client.plaintext.boxtitle=Plain-text (with STARTTLS) connections
 ssl.settings.client.plaintext.info=Openfire can accept plain-text connections, which, depending on the policy that is configured here, can be upgraded to encrypted connections (using the STARTTLS protocol).
 ssl.settings.client.plaintext.label_enable=Enabled
-ssl.settings.client.legacymode.boxtitle=Encrypted (legacy-mode) connections
-ssl.settings.client.legacymode.info=Connections of this type are established using encryption immediately (as opposed to using STARTTLS). This type of connectivity is commonly referred to as the "legacy" method of establishing encrypted communications.
-ssl.settings.client.legacymode.label_enable=Enabled
```

Comment:
These got removed but not replaced which gives missing strings on connection-settings-socket-c2s.jsp